src/main/java/data/CameraTimeline.java

```diff
@@ -12,6 +14,9 @@
     // The camera that is associated with this timeline.
     @Getter @Setter
     private Camera camera;
+    
+    @Getter
+    private ArrayList<CameraShot> shots;
```

Comment:
The TimeLine class already contains a list of shots, can you use that? If not you should remove that and also add a list of DirectorShots to the DIrectorTimeline